A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -185,13 +188,36 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
 
         this.A32NXCore.update();
 
+        this.updateMCDU();
+
         this.updateAutopilot();
 
         this.updateScreenState();
 
         this.updateGPSMessage();
     }
 
+    /**
+     * Checks whether INIT page B is open and an engine is being started, if so:
+     * The INIT page B reverts to the FUEL PRED page 15 seconds after the first engine start and cannot be accessed after engine start.
+     */
+    updateMCDU() {
+        if (this.isAnEngineOn()) {
+            if (!this.initB) {
+                this.initB = true;
+                if (this.getTitle() === this.initBTitle) {
+                    setTimeout(() => {
+                        if (this.getTitle() === this.initBTitle) {
+                            CDUFuelPredPage.ShowPage(this);
+                        }
+                    }, 15000);
```

Comment:
Wouldn't this be called everytime updateMCDU() is called? thus starting several timeouts over and over?

Reply:
No it only creates one timeout.